### PR TITLE
[Merged by Bors] - feat: monovarying functions are simultaneously monotone

### PR DESCRIPTION
**Pull request by YaelDillies**
From LeanCamCombi

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Monotone/MonovaryOrder.lean [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2022 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel, Yury Kudryashov
```
Comment (urkud) [inline Mathlib/Order/Monotone/MonovaryOrder.lean:4]: I didn't author this file!

Comment (urkud) [reply Mathlib/Order/Monotone/MonovaryOrder.lean:4]: Otherwise LGTM. Thanks!
bors d+

Comment (YaelDillies) [reply Mathlib/Order/Monotone/MonovaryOrder.lean]: Uh, not sure what happened here!

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**